px_metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "7.2.0",
+    "version": "7.3.0",
     "supported_features": [
```

Comment:
We don't have in px_metadata a record for CI (credentials_extraction or login_credentials_extraction or credentials_intelligence)?

Reply:
I see you did add it to the other PR, what about here?

Reply:
I see you did add it to the other PR, what about here?